src/main.c

```diff
@@ -632,6 +632,18 @@ void vb_statusbar_update(Client *c)
 
     statusbar_update_downloads(c, status);
 
+    if ( c->config.statusbar_show_settings ) {
+        /* show js, cookies, and local storage status */
+
+        Setting *js = g_hash_table_lookup(c->config.settings, "scripts");
+        Setting *cookies = g_hash_table_lookup(c->config.settings, "cookie-accept");
+        Setting *local_storage = g_hash_table_lookup(c->config.settings, "html5-local-storage");
+
+        g_string_append_printf(status, " js: %s |", js->value.b ? "on" : "off");
+        g_string_append_printf(status, " cookies: %s |", cookies->value.s);
+        g_string_append_printf(status, " storage: %s", local_storage->value.b ? "on" : "off");
```

Comment:
Might it possible to allow the user to define these values by config.h file. I Think this is a nice feature to see which settings are set. But users might want so se other values here.
I think it would be nice to define the settings to be read and the visualization to compile time. But I'm not sure if this would be easy possible with some kind of array of structs ore something similar.

Reply:
Hmm it is possbile only if we limit ourselves to one kind of value: ints, strings, or booleans.
Because as you can see, cookies have three possible options, while the others are just booleans.
I'll think of a way to handle that and see how it goes.

Reply:
Alright I got it done, let me know if you like the change so I can update the man doc :)